Fail to build lan78xx ethernet driver on Raspberry Pi OS lite (64 bit)

**Describe the bug**
On a fresh install of Raspberry Pi OS lite (64 bit), I'm following this guide:
https://wiki.seeedstudio.com/Dual-Gigabit-Ethernet-Carrier-Board-for-Raspberry-Pi-CM4/

> Once you flash the Raspberry Pi OS onto the CM4 module, you will get the full Gigabit speed (1Gbps) only on the Ethernet port connected to the Gigabit Ethernet PHY of the CM4 module which is based on Broadcom BCM54210PE (right-side port). The port which is connected to the Microchip's LAN7800 USB 3.0 to GbE (Gigabit Ethernet Bridge) (left-side port), will not provide the full speed of 1Gbps, but rather a much reduced speed. This is because the lan78xx driver in the raspberry core is not up to date.

Building the driver fails with the following error:
```
root@seed:/home/pi/git/seeed-linux-dtoverlays# ./scripts/cm4_lan7800.sh
/bin/bash: warning: setlocale: LC_ALL: cannot change locale (en_US.UTF-8)
Installed: /usr/src/linux-headers-5.15.32-v8+
make: Entering directory '/usr/src/linux-headers-5.15.32-v8+'
  CC [M]  /home/pi/git/seeed-linux-dtoverlays/modules/lan7800/lan78xx.o
/home/pi/git/seeed-linux-dtoverlays/modules/lan7800/lan78xx.c: In function 'lan78xx_skb_return':
/home/pi/git/seeed-linux-dtoverlays/modules/lan7800/lan78xx.c:3511:20: error: 'GRO_DROP' undeclared (first use in this function)
 3511 |  if (gro_result == GRO_DROP)
      |                    ^~~~~~~~
/home/pi/git/seeed-linux-dtoverlays/modules/lan7800/lan78xx.c:3511:20: note: each undeclared identifier is reported only once for each function it appears in
make[1]: *** [scripts/Makefile.build:277: /home/pi/git/seeed-linux-dtoverlays/modules/lan7800/lan78xx.o] Error 1
make: *** [Makefile:1868: /home/pi/git/seeed-linux-dtoverlays/modules/lan7800] Error 2
make: Leaving directory '/usr/src/linux-headers-5.15.32-v8+'
Build failed: lan7800
```

**To Reproduce**
On Raspberry Pi OS lite (64 bit):
```
> git clone https://github.com/Seeed-Studio/seeed-linux-dtoverlays.git
> cd seeed-linux-dtoverlays/
> sudo ./scripts/cm4_lan7800.sh
```

**Additional context**
```
root@seed:/home/pi/git/seeed-linux-dtoverlays# cat /etc/debian_version 
11.4
root@seed:/home/pi/git/seeed-linux-dtoverlays# uname -a 
Linux seed 5.15.32-v8+ #1538 SMP PREEMPT Thu Mar 31 19:40:39 BST 2022 aarch64 GNU/Linux
```

--- a/modules/lan7800/lan78xx.c
+++ b/modules/lan7800/lan78xx.c
@@ -3508,8 +3508,6 @@ static void lan78xx_skb_return(struct lan78xx_net *dev, struct sk_buff *skb)
 
 	gro_result = napi_gro_receive(&dev->napi, skb);
 
-	if (gro_result == GRO_DROP)
-		netif_dbg(dev, rx_err, dev->net, "GRO packet dropped\n");
 }
 
 static int lan78xx_rx(struct lan78xx_net *dev, struct sk_buff *skb,

